crates/tauri-bundler/src/bundle/windows/sign.rs

```diff
@@ -214,7 +214,9 @@ pub fn sign_custom<P: AsRef<Path>>(
   let output = cmd.output_ok()?;
 
   let stdout = String::from_utf8_lossy(output.stdout.as_slice()).into_owned();
-  log::info!("{:?}", stdout);
```

Comment:
I believe `log::info!("{stdout}");` would just work

Reply:
that would look like this 
<img width="1096" height="134" alt="grafik" src="https://github.com/user-attachments/assets/1b38b2a3-a50b-4168-acea-1952942ed453" />

i wouldn't mind that either especially since idk what other tools may output

Reply:
stop it with your extra wishes